pages/knowledge/screen-readers/relevant-combinations/README.md

```diff
@@ -14,9 +14,16 @@ As web developers, we're aware of how hard it is to support all relevant devices
 
 It is impossible to develop and test for dozens of possible combinations of browsers and screen readers. We have had good experiences with taking the following pragmatic approach, namely aiming at the most widely used combinations (newest stable versions):
 
-- JAWS with Internet Explorer (IE)
-- JAWS with Firefox (FF)
-- NVDA with FF
+### Mobile screen readers
+
+- VoiceOver/iOS with Safari
+- TalkBack with Chrome
+
+### Desktop/Laptop screen readers
+
+- JAWS with Chrome
+- NVDA with Firefox (FF)
+- NVDA with Chrome
```

Comment:
Although market share is very close to NVDA+FF, we neglect NVDA+Chrome (as they both are pretty good and stable, so we expect them to work well together - and we have to keep the amount of supported combinations manageable).

So please remove this combination from the list.